### ISSUE TITLE
 Error: Unable to resolve module AccessibilityInfo

### ISSUE THREAD

**Issue by AmrutaPawar025** [awslabs/aws-mobile-appsync-sdk-js#756]
error: Error: Unable to resolve module AccessibilityInfo from D:\Projects\atm-app\frontEnd1\mypoc\node_modules\aws-appsync-react\node_modules\react-native\Libraries\react-native\react-native-implementation.js: AccessibilityInfo could not be found within the project or in these directories:
  node_modules\aws-appsync-react\node_modules\react-native\node_modules
  node_modules\aws-appsync-react\node_modules
  node_modules
  17 | const ReactNative = {
  18 |   // Components
> 19 |   get AccessibilityInfo() { return require('AccessibilityInfo'); },
     |                                             ^
  20 |   get ActivityIndicator() { return require('ActivityIndicator'); },
  21 |   get ART() { return require('ReactNativeART'); },
  22 |   get Button() { return require('Button'); },
    at ModuleResolver.resolveDependency (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:139:15)
    at DependencyGraph.resolveDependency (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\node-haste\DependencyGraph.js:277:43)
    at Object.resolve (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\lib\transformHelpers.js:169:21)
    at Graph._resolveDependencies (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\DeltaBundler\Graph.js:473:35)
    at Graph._processModule (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\DeltaBundler\Graph.js:261:38)
    at async Graph._addDependency (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\DeltaBundler\Graph.js:372:20)
    at async Promise.all (index 2)
    at async Graph._processModule (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\DeltaBundler\Graph.js:322:5)
    at async Graph._addDependency (D:\Projects\atm-app\frontEnd1\mypoc\node_modules\metro\src\DeltaBundler\Graph.js:372:20)
    at async Promise.all (index 0)
